packages/instant/src/index.umd.ts

```diff
@@ -5,52 +5,98 @@ import * as ReactDOM from 'react-dom';
+    }
+    if (!_.isUndefined(config.additionalAssetMetaDataMap)) {
+        assert.isValidAssetMetaDataMap('additionalAssetMetaDataMap', config.additionalAssetMetaDataMap);
     }
-    if (!_.isUndefined(props.additionalAssetMetaDataMap)) {
-        assert.isValidAssetMetaDataMap('props.additionalAssetMetaDataMap', props.additionalAssetMetaDataMap);
+    if (!_.isUndefined(config.defaultAssetBuyAmount)) {
+        assert.isNumber('defaultAssetBuyAmount', config.defaultAssetBuyAmount);
     }
-    if (!_.isUndefined(props.defaultAssetBuyAmount)) {
-        assert.isNumber('props.defaultAssetBuyAmount', props.defaultAssetBuyAmount);
+    if (!_.isUndefined(config.networkId)) {
+        assert.isNumber('networkId', config.networkId);
     }
-    if (!_.isUndefined(props.networkId)) {
-        assert.isNumber('props.networkId', props.networkId);
+    if (!_.isUndefined(config.availableAssetDatas)) {
+        assert.areValidAssetDatas('availableAssetDatas', config.availableAssetDatas);
     }
-    if (!_.isUndefined(props.availableAssetDatas)) {
-        assert.areValidAssetDatas('availableAssetDatas', props.availableAssetDatas);
+    if (!_.isUndefined(config.onClose)) {
+        assert.isFunction('onClose', config.onClose);
     }
-    if (!_.isUndefined(props.onClose)) {
-        assert.isFunction('props.onClose', props.onClose);
+    if (!_.isUndefined(config.zIndex)) {
+        assert.isNumber('zIndex', config.zIndex);
     }
-    if (!_.isUndefined(props.zIndex)) {
-        assert.isNumber('props.zIndex', props.zIndex);
+    if (!_.isUndefined(config.affiliateInfo)) {
+        assert.isValidAffiliateInfo('affiliateInfo', config.affiliateInfo);
     }
-    if (!_.isUndefined(props.affiliateInfo)) {
-        assert.isValidAffiliateInfo('props.affiliateInfo', props.affiliateInfo);
+    if (!_.isUndefined(config.provider)) {
+        assert.isWeb3Provider('provider', config.provider);
     }
-    if (!_.isUndefined(props.provider)) {
-        assert.isWeb3Provider('props.provider', props.provider);
+    if (!_.isUndefined(config.shouldDisablePushToHistory)) {
+        assert.isBoolean('shouldDisablePushToHistory', config.shouldDisablePushToHistory);
     }
     assert.isString('selector', selector);
-    const appendToIfExists = document.querySelector(selector);
-    assert.assert(!_.isNull(appendToIfExists), `Could not find div with selector: ${selector}`);
-    const appendTo = appendToIfExists as Element;
-    const injectedDiv = document.createElement('div');
-    injectedDiv.setAttribute('id', INJECTED_DIV_ID);
-    injectedDiv.setAttribute('class', INJECTED_DIV_CLASS);
-    appendTo.appendChild(injectedDiv);
-    const instantOverlayProps = {
-        ...props,
-        onClose: () => {
-            appendTo.removeChild(injectedDiv);
-            if (!_.isUndefined(props.onClose)) {
-                props.onClose();
+    // Render instant and return a callback that allows you to remove it from the DOM.
+    const renderInstant = () => {
```

Comment:
`render` has gotten pretty big.  I would prefer for this function to be defined as it's own top-level function instead of defined in here